data_models/migrations/0024_populate_show_digas_id.py

```diff
@@ -31,13 +39,24 @@ def populate_digas_id(apps, schema_editor):
                 continue
 
             # We found something!
-            show.digas_id = digas_id_by_on_demand_link[sod_url]
+            digas_id = digas_id_by_on_demand_link[sod_url]
+            if digas_id in used_digas_ids: 
+                # Alert the user and don't update this show.
+                log.exception('Duplicate digas_id encountered for {show}.' + \
```

Comment:
Same as below, `log.exception` is for exception handlers

```suggestion
                log.error('Duplicate digas_id encountered for {show}.' + \
```